crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -394,17 +350,14 @@ pub mod tests {
     #[test]
     fn test_qml_metadata() {
         let item: ForeignTypeIdentAlias = parse_quote! {
-            #[cxx_qt::qobject(qml_uri = "foo.bar", qml_version = "1.0")]
+            #[cxx_qt::qobject(qml_element)]
             type MyObject = super::MyObjectRust;
         };
```

Comment:
I was thinking of having the syntax be like:
```rust
#[cxx_qt::qobject]
#[qml_element]
#[qml_singleton]
type MyObject = super::MyObjectRust;
```

but this was easier to implement and also makes sense, so :shrug:

Reply:
I think eventually the syntax will be either
```rust
#[qml_element]
#[qml_singleton]
type MyObject = super::MyObjectRust;
```
or
```rust
#[qobject]
#[qml_element]
#[qml_singleton]
type MyObject = super::MyObjectRust;
```

But what you have at the moment is fine as a stepping stone, we can sort out exact the syntax in follow ups :-)